atlassian/bitbucket/cloud/__init__.py

```diff
@@ -3,13 +3,16 @@
 from .base import BitbucketCloudBase
 from .workspaces import Workspaces
 from .repositories import Repositories
+from .const import CONF_TIMEFORMAT
+from datetime import datetime
 
 
 class Cloud(BitbucketCloudBase):
     def __init__(self, url, *args, **kwargs):
         kwargs["cloud"] = True
         kwargs["api_root"] = None
         kwargs["api_version"] = "2.0"
+        kwargs["timeformat_func"] = kwargs.pop("timeformat_func", lambda ts: datetime.strptime(ts, CONF_TIMEFORMAT))
```

Comment:
This block shall be moved to `atlassian/bitbucket/base.py` because the attribute is stored there and the server shall use the same value.
```python
kwargs["timeformat_func"] = kwargs.pop("timeformat_func", lambda x: x)
```